internal/sync.go

```diff
@@ -402,6 +402,10 @@ func (s *syncGSuite) SyncGroupsUsers(query string) error {
 	log.Info("syncing changes")
 	// delete aws users (deleted in google)
 	log.Debug("deleting aws users deleted in google")
+	if !checkUserDeletionThreshold(delAWSUsers) {
```

Comment:
Maybe this and the other function are redundant as just having just the if here is enough

```go
if len(delAWSUsers) > deletionThreshold { ... }
```